docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -182,7 +182,6 @@ Do you want to assign a burner wallet to a slot? (yes/no):
 ```
 
 Enter yes and then enter your burner wallet address when prompted.
```

Comment:
We should remove this line, too; we never ask for the burner wallet, only the holding address private key.

Reply:
Noted. Done, I raised a PR; please review it.